### DISCUSSION TITLE
Is there any way to show the markdown syntax while keeping the formatting?

### DISCUSSION THREAD

**Discussion by tylershuster** [outline/rich-markdown-editor#297] [General] [answered]
I mean like this:

<img width="304" alt="image" src="https://user-images.githubusercontent.com/3999320/94623235-94b9f000-0268-11eb-82e2-dcf24e471b8b.png">

Happy to provide more explanation if needed

**Comment by tommoor** [comment, marked as answer]
This project doesn't provide that sort of display right now, it could be interesting to make it a flag/prop but I'd imagine it would be quite a lot of effort.

**Reply by tylershuster** [reply]
Thanks; I appreciate the quick response.
